src/clj/rems/workflow/dynamic.clj

```diff
@@ -272,7 +272,7 @@
 (defn- applicant-error
   [application cmd]
   (when-not (= (:actor cmd) (:applicantuserid application))
-    {:errors [:forbidden]}))
+    {:errors [{:key :forbidden}]}))
```

Comment:
why `:key` and not `:dynamic.error/type` or something?

Reply:
It is fed directly to the localization. That's the format used in field errors already.

Reply:
I remembered wrong and it's `:type` elsewhere already so I'll just use that. If we want we can sometime refactor it to e.g. `:error/type`